taurus/entity/bounds/tests/test_integer_bounds.py

```diff
@@ -21,24 +18,9 @@ def test_incompatible_types():
     """Make sure that incompatible types aren't contained or validated."""
     int_bounds = IntegerBounds(0, 1)
 
-    assert not int_bounds.validate(NominalReal(0.0, ''))
-
     assert not int_bounds.contains(RealBounds(0.0, 1.0, ''))
 
 
-def test_validate():
-    """Test validation logic."""
-    int_bounds = IntegerBounds(0, 2)
-
-    assert int_bounds.validate(NominalInteger(0))
-    assert int_bounds.validate(NominalInteger(2))
-    assert not int_bounds.validate(NominalInteger(3))
-
-    assert not int_bounds.validate(UniformInteger(0, 3))
-    with pytest.raises(TypeError):
-        int_bounds.validate(0)
-
-
```

Comment:
I disagree with removing this

Reply:
This isn't changing the test coverage; its a consequence of removing the `validate` method.